swift-sdk/Constants.swift

```diff
@@ -204,6 +204,11 @@ enum JsonKey {
         static let userInterfaceIdiom = "userInterfaceIdiom"
     }
     
+    enum Embedded {
+        static let packageName = "packageName"
+        static let sdkVersion = "SDKVersion"
```

Comment:
Why within Embedded?

Reply:
This is similar to the way in-app is defined.

Reply:
I wanted to give it its own namespace within the constants. Previously it was using the in-app's constants.